tamr_unify_client/client.py

```diff
@@ -144,12 +144,12 @@ def set_auth_cookie(self):
             "./instance:login",
             json={"username": self.auth.username, "password": self.auth.password},
         )
-        if r.status_code != 200:
-            raise Exception  # TODO: Replace with sensible handling
-        auth_token = r.json()["token"]
-        self.session.cookies.set("authToken", auth_token)  # TODO: Set domain
-        # Clear session auth.  It is still accessible as self.auth if needed
-        self.session.auth = None
+        # Only continue if request is successful
+        if r.ok:
```

Comment:
alternative: I slightly prefer a guard clause here. up to you 😄 

```suggestion
        if !r.ok: return
```

less indentation and keeps it clear what the "happy path" is

Reply:
I'm good with that.  For some reason my brain wasn't happy with adding a return to a method, but that's obviously fine.